app/views/alchemy/admin/attachments/_archive_overlay.html.erb

```diff
@@ -1,21 +1,18 @@
 <div id="overlay_toolbar">
-  <div class="button_with_label">
-    <%= link_to_dialog(
-        render_icon('upload'),
-        alchemy.new_admin_attachment_path(
-          content_id: @content.blank? ? nil : @content.id,
-          swap: @swap,
-          options: @options.to_json
-        ),
-        {
-          title: Alchemy.t(:upload_file),
-          size: '540x550'
-        },
-        title: Alchemy.t(:upload_file),
-        class: 'icon_button'
+  <% if can? :create, Alchemy::Attachment %>
+    <%= render 'alchemy/admin/partials/upload_button',
```

Comment:
Maybe we should move the uploader partials into its own folder? `alchemy/admin/uploader/button`